crates/rattler_installs_packages/src/python_env/distribution_finder.rs

```diff
@@ -55,6 +55,31 @@ pub enum FindDistributionError {
     FailedToParseWheelTag(String),
 }
 
+/// Locates the python distributions (packages) that have been installed in the specified directory.
+///
+/// When packages are installed in a venv they are installed in specific directories. Use the
+/// [`find_distributions_in_venv`] if you don't want to deal with determining the proper directory.
+///
+/// Any path in the results is relative to `dir`.
+pub fn find_distributions_in_directory(
+    dir: &Path,
```

Comment:
Maybe rename this to `search_dir` or something?

Reply:
Done!